packages/server/api/src/app/database/migrations/1746627735717-AddStepIdToExistingTemplates.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { logger } from '@openops/server-shared';
+import { openOpsId } from '@openops/shared';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddStepIdToExistingTemplates1746627735717
```

Comment:
why do we need this as part of the migration?
the FE should generate the ids when using a template